spec/controllers/api/scores_controller_spec.rb

```diff
@@ -104,4 +104,17 @@
       expect(Score.count).to eq score_count
     end
   end
+
+  describe 'GET user scores' do
+    it 'should return all scores for a user' do
+      get api_path(@user2.id)
+
+      expect(response).to have_http_status(:ok)
+      response_hash = JSON.parse(response.body)
+      scores = response_hash['scores']
+
+      expect(scores.size).to eq 2
+      expect(scores[0]['user_name']).to eq 'User2'
```

Comment:
If someone changes the user's name, this test will fail, but it's not because some application logic was changed, but because the test was changed.

Instead of testing against a fixed string `User2`, use `@user2.name`